### DISCUSSION TITLE
How to compare values ​​with NULL without getting an error?

### DISCUSSION THREAD

**Discussion by getitsoIved** [josdejong/mathjs#3129] [Q&A]
Hi!

We use expressions to compare a variable with a value ( a > 10). The value of a variable may be Null or undefined. At the same time, the formula breaks down and an error occurs; I would like Null > 10 to result in False.

Is it possible to define your own constant in the library that will work in accordance with the desired result?

**Comment by josdejong** [comment]
I don't think there is an easy way to alter the behavior of null. It may be interesting to look into implementing explicit support for `null` in all relational functions/operators (equal, unequal, larger, smaller, etc).

**Comment by dvd101x** [comment]
You could also import the new signatures to the function larger in your local instance.
```js
math.import({
    'larger': math.typed('larger',{
        'undefined | null | any, undefined | null | any' : () => false
    })
}) 
```
After that you could evaluate
```jl
[null, 13, 2, undefined, NaN] > 3      # [false, true, false, false, false]
````
The current functions can deal with NaN, you could also use that to your benefit for edge cases.
